src/test/resources/cucumber/features/portalAdmin/DARTS_admin.feature

```diff
@@ -966,3 +966,26 @@ Feature: Admin
     And I see "Created by" in the same row as "System"
     And I see "Date last modified" in the same row as "22 Aug 2024 at 16:16:55"
     And I see "Last modified by" in the same row as "System"
+
+    @DMP-2731
+    Scenario: Transcript files marked for deletion
+      When I am logged on to the admin portal as an ADMIN user
+      And I click on the "File deletion" link
+      And I see "Audio files" in the same row as "Transcripts"
+      Then I click on the "Transcripts" sub-menu link
+      And I see "Transcripts" on the page
+      Then I verify the HTML table contains the following values
+      | Transcript ID| Case ID               | Hearing date| Courthouse| Courtroom         | Marked by  | Comments                             | *IGNORE* |
+      | 8149         | DMP-3364-Disable-User | 24 Sep 2024 | Swansea   | OURTROOM-DMP-3364 | Darts Admin| Public interest immunity\nTest\nTest | Delete   |
+
+      Then I press the "Delete" button
+      And I see "You cannot delete this transcript file" on the page
+      And I see "You cannot delete a transcript file that you marked for deletion." on the page
+      And I see "Another administrator will need to check the file can be deleted and can complete this step." on the page
+      And I see "Go back" on the page
+
+      Then I click on the "Go back" link
+      And I see "Transcripts" on the page
+      And I verify the HTML table contains the following values
```

Comment:
Do we not want to include actually deleting the transcription in this test, as another admin user?

Reply:
Deleting the transcription required to approved by another admin user and only one super admin we have in automation

Reply:
workaround might be to show that the admin user cant delete their own
then update the record to make it appear that a different admin user had deleted
then go back to our admin user & delete